### ISSUE TITLE
Need support for the latest version of Starlette

### ISSUE THREAD

**Issue by russian-developer** [python-gino/gino-starlette#20]
I've found that the latest release of gino-stralette (0.1.1) doesn't support the latest version of Starlette (0.14.2).
It requires >0.14.0.

**Comment by frall88** [comment]
I faced with the issue in my fastapi app when trying to update fastapi version due to [security issue](https://github.com/samuelcolvin/pydantic/security/advisories/GHSA-5jqp-qgf6-3pvh):
```
 Because no versions of fastapi match >0.65.1,<0.66.0
   and fastapi (0.65.1) depends on starlette (0.14.2), fastapi (>=0.65.1,<0.66.0) requires starlette (0.14.2).
  Because gino-starlette (0.1.1) depends on starlette (>=0.13.0,<0.14.0)
   and no versions of gino-starlette match >0.1.1,<0.2.0, gino-starlette (>=0.1.1,<0.2.0) requires starlette (>=0.13.0,<0.14.0).
  Thus, fastapi (>=0.65.1,<0.66.0) is incompatible with gino-starlette (>=0.1.1,<0.2.0).
  And because gino[pg,starlette] (1.0.1) depends on gino-starlette (>=0.1.1,<0.2.0)
   and no versions of gino match >1.0.1,<2.0.0, fastapi (>=0.65.1,<0.66.0) is incompatible with gino[pg,starlette] (>=1.0.1,<2.0.0).
  So, because marketplace depends on both fastapi (^0.65.1) and gino[pg,starlette] (^1.0.1), version solving failed.
```

**Comment by Chaostheorie** [comment]
Will try to add support with a PR on the weekend. Though I would be happy if someone, who actually uses this extension and as such has experience with, would be wiling to pick this up and provide further support. It has been long, since I've worked with starlette/ fastapi.

**Comment by wwwjfy** [comment]
fixed via #24
